tasks/change-granule-collection-pg/src/index.ts

```diff
@@ -77,9 +81,12 @@ function validateGranule(granule: ApiGranuleRecord): granule is ValidGranuleReco
 }
 
 function validateConfig(config: EventConfig): ValidEventConfig {
```

Comment:
I know this function is super simple, but if we're unit testing all functions, we should have a simple test for it.